frontend/src/widgets/Overview.css

```diff
@@ -9,8 +9,7 @@
 }
 
 .smart-hub--overview-data {
-  margin-left: -20px;
-  margin-bottom: -20px;
+  margin-left: -36px !important;
```

Comment:
is there a way to be more specific without ```!important```?

Reply:
I found that this was the only way I could get that to work. Not ideal, since !important is a last resort technique. Perhaps once the layout changes, e.g. with addition of the welcome we could revisit.